Package.swift

```diff
@@ -53,7 +57,33 @@ let package = Package(
                 .when(platforms: [.iOS, .macCatalyst, .macOS, .tvOS, .appCheckVisionOS])
               ),
             ]),
-    // TODO(andrewheard): Add unit test targets after removing Firebase dependencies.
+    .testTarget(
+      name: "AppCheckCoreUnit",
+      dependencies: [
+        "AppCheckCore",
+        .product(name: "OCMock", package: "ocmock"),
+      ],
+      path: "AppCheckCore/Tests",
+      exclude: [
+        // Swift tests are in the target `AppCheckCoreUnitSwift` since mixed language targets are
+        // not supported (as of Xcode 14.3).
+        "Unit/Swift",
```

Comment:
Can you make a separate .testTarget for Swift?

Reply:
The Swift `testTarget` is below: `AppCheckCoreUnitSwift`

Reply:
I see. Thanks.